common/service/pldm/pldm_monitor.c

```diff
@@ -431,6 +431,10 @@ uint8_t pldm_send_platform_event(uint8_t event_class, uint16_t id, uint8_t ext_c
 		return pldm_send_effecter_event_message(event_receiver_info_p->mctp_inst_p,
 							event_receiver_info_p->ext_params, id,
 							ext_class, event_data, event_data_length);
+	case PLDM_OEM_EVENT_START ... PLDM_OEM_EVENT_END:
```

Comment:
This is a GCC extension that we should probably refrain from using.

Reply:
Ok, thanks for the suggestion, will avoid using the extension like this in the future.